### DISCUSSION TITLE
Локализация объектов звездного поля и планет

### DISCUSSION THREAD

**Discussion by SirMaxBlack** [SnakeSel/Empyrion-ru#6] [Show and tell]
Добрый день!)

Добавлю немного помощи в переводе, со своей стороны, если вы не против.
Создал в гугле таблицу с локализацией обьектов, указанных в заголовке темы.

![image](https://github.com/SnakeSel/Empyrion-ru/assets/6567430/79807bd6-7f6d-4abe-8db2-75ab1d5f166b)

Методом "научного тыка" выяснил, что нужно писать перевод предметов в файл Localization.csv
Причем, почему-то, в оба сразу, иначе перевод не работает/работает через раз.

По мере игры нахожу обьекты без перевода. Добавляю их в таблицу и перевожу.
Перевожу с помощью яндекс+гугл переводчика и результат чапстично обрабатываю сам, если звучит нелогично в переводе.
В этом столбце уже готовый текст для переноса в файл :
![image](https://github.com/SnakeSel/Empyrion-ru/assets/6567430/dd7dfbe8-e43e-4842-ac7b-1fe973073bd3)

Сама таблица. Сейчас у вошедших стоит режим "чтения". Если надо, могу выдать права на редактирование.
Чтобы не создавать путаницу для вас, буду создавать новые вкладки в таблице с датами добавления новых обьектов, после вашей проверки и занесения в релизную ветку репозитория.

https://docs.google.com/spreadsheets/d/1otwVYXqG71eW8vFh2UTgc7eDNxT60ugY8iWdwryYnfo/edit?usp=sharing

Если есть замечания/предложения/посыл нахер =D, скажите плз)

**Comment by SirMaxBlack** [comment]
К сожалению чет не получается перевести тут:
![image](https://github.com/SnakeSel/Empyrion-ru/assets/6567430/e089f0be-60bc-43e1-ab18-020410a78451)


Хотя нужный текст нашел.
Отдельное извращение в описании самих планет. Т.к. там есть перевод строки. В тихую впихнуть \n не прокатило
![image](https://github.com/SnakeSel/Empyrion-ru/assets/6567430/6c826d64-14b6-4e95-a72d-62af4438d2ba)

**Reply by SnakeSel** [reply]
Тут текст прописан в файле. Его через locales не поменять. Только сам файл редактировать

**Comment by SnakeSel** [comment]
Внес изменения в файл локализации RE с небольшими правками

**Comment by All4X4** [comment]
Местами странный перевод. Местами нужно искать синонимы чтоб сократить текст, в частности касаемых маркеров. Иначе, в игре, будет каша. Я вообще не сторонник перевода подобных названий ("имён собственных" и прочих), их часто используют по квестам, а потом гадай куда тебя послали и как это однозначно перевели в ПДА/диалогах и локал-файле.

Оговорюсь, всё ниже написанное: это сугубо ИМХО, без нападок. Просмотрел файл "по диагонали"....

**Agri-Station** - Агростанция (Агро-станция) от слова аграрный, гораздо короче "Сельскохозяйственный", а это важно. )

**Crashed Titan (Rear)** - Разбитый Титан (задняя) (зад), но не "конец", можно даже "корма" "Корма́ — задняя часть корпуса военного корабля "

**Drone Base** - Всегда была и будет "Дрон База", слово "беспилотники" длинное, а слово "дрон" уже устоялось как в игре, так и в реале.

Лично меня коробит в игре, при переводе, слово "горнодобывающий(ая)", какие горы в космосе? Да и длинное. Я вечно с ним ломаю голову...

**Mining Tower** - Шахтная Башня    это так в реале обзывают

**Oxygen Factory** - Кислородный завод или фабрика, тут нет слова "станция"

**Lava Tubes** - Лавовые Трубка   "Лавовые трубка - полости в лавовых потоках, вытянутые в виде коридоров. Такие каналы получаются при неравномерном остывании текущей со склонов вулкана лавы."

**Mining Rig** - это буровая, тут нет слова "бур"

**Brewery** - это просто "пивоварня" без слова "завод"

**Bazaar** - Базар, авторы RE вставили это слово для колорита быта Талон, зачем его переиначивать? )

**Old Tank Station** - Старая резервуарная станция... тут не подходит "Старая танковая станция" ))) Всё что в игре со словом "tank"- это про баки/резервуары. Емнип, есть исключение в оружии, но это не точно.

**Asteroid Mining Hub** - Центр по добыче астероидов (это короче, без "полезных ископаемых")

И не переводите название "фирм/производителей/фракций и подфракций, собственное название станций, кораблей" и тп, это вызовет путаницу.

**Reply by SirMaxBlack** [reply]
Оговорюсь, я производил перевод через онлайн переводчики.
Если нужно, могу расшарить доступ к таблице, чтобы вы смогли внести коррективы.

ИМХО. Если можно перевести, чтобы было понятно хоть как-то, это лучше, чем вообще не переводить. У меня знания английского околонулевые, учитывая, что мой родной язык-казахский. И когда мне в квесте говорят, сходи на такой-то завод и приобрети "то-то", я зависаю и просто забиваю на квест т.к. большенство названий таких обьектов было на иностранном.

**Reply by SirMaxBlack** [reply]
Вот одна из ситуаций, в качестве примера.

![image](https://github.com/SnakeSel/Empyrion-ru/assets/6567430/fd582a15-25bb-4768-9674-3e80fc0c916a)

Вот что выдают переводчики:
![image](https://github.com/SnakeSel/Empyrion-ru/assets/6567430/fc4d6a92-2d72-486f-bf37-3283493cb433)
![image](https://github.com/SnakeSel/Empyrion-ru/assets/6567430/1d6559a9-d9f4-47ce-9711-d32eee6f4830)


В игре такой предмет еще не встречал, поэтому даже предположить не могу, правильно ли переведено или нет. И если выйдет так, что предмет в игре переведен а в задании нет, то крайне мала вероятность, что я смогу ее найти.
Предполагаю, что и дальше будут проблемы в переводе т.к. переводят и корректируют текст разные люди.

**Reply by All4X4** [reply]
В принципе правильно переведено, сурово в лоб. Только Варон корень. Вроде как растение, типа мандрагоры. ))) Надо в игру зайти чтоб точнее глянуть в айтем меню, на предмет ошибки.

**Reply by SirMaxBlack** [reply]
![image](https://github.com/SnakeSel/Empyrion-ru/assets/6567430/1254e12d-3716-4312-8127-eba19720ab73)

**Comment by All4X4** [comment]
> К сожалению чет не получается перевести тут: ![image](https://private-user-images.githubusercontent.com/6567430/327970345-e089f0be-60bc-43e1-ab18-020410a78451.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUyNjc3MjUsIm5iZiI6MTcxNTI2NzQyNSwicGF0aCI6Ii82NTY3NDMwLzMyNzk3MDM0NS1lMDg5ZjBiZS02MGJjLTQzZTEtYWIxOC0wMjA0MTBhNzg0NTEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUwOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MDlUMTUxMDI1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YTA5Zjg5Zjk0NDcwODZkMWVmMzYxNTdhZDgzZDZhZTg1MmE0NDBjZjA1NGYzNTYzMDdhMTg5ZjZkZGVhODkxMSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.f11AIGNRXsn7oCntVTUnlopUEICQB1NowciJB00utvc)
> 
> Хотя нужный текст нашел. Отдельное извращение в описании самих планет. Т.к. там есть перевод строки. В тихую впихнуть \n не прокатило ![image](https://private-user-images.githubusercontent.com/6567430/327970407-6c826d64-14b6-4e95-a72d-62af4438d2ba.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUyNjc3MjUsIm5iZiI6MTcxNTI2NzQyNSwicGF0aCI6Ii82NTY3NDMwLzMyNzk3MDQwNy02YzgyNmQ2NC0xNGI2LTRlOTUtYTcyZC02MmFmNDQzOGQyYmEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUwOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MDlUMTUxMDI1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OTY0M2MyOGMwNmIyODFiYWRiYWMyNjAyZTI3NTAwMGI2OTM4ZWM4ZDVlZWE0MWI4MTk0MmQxNzVlZmVhNTZhZiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.6p_BPD60n6bCZwdoNCkyEp5pQ3IiaYaFOHY0Z-w_VEU)

На 1 скрине это GalaxyConfig.ecf, для себя можно перевести, а для масс лучше не надо.

По игровым полям тоже можно, муторно, но можно.  Главное придерживаться синтаксиса ямл,  иначе будут ошибки/краши.

![изображение](https://github.com/SnakeSel/Empyrion-ru/assets/164763695/a1927ae6-aebf-4baf-8dc9-0958435256b0)

**Reply by SirMaxBlack** [reply]
Скорее всего будет не хорошим вариантом переводить эти файлы. Иначе, при выходе обновы, все придется делать заново вручную(

**Reply by All4X4** [reply]
Игровые поля сейчас редко правятся (вообще не правятся) в RE. Но для массового перевода я бы не советовал. Мало кто будет менять ради перевода эти файлы, да и замучаешься потом объяснять куда и что заливать, а зальют куда непопадя - 146% инфа. Как и не стоит переводить для всех файлы ecf. Кроме этого, любая ошибка синтаксиса и поимеем краши.

Для таких случаев есть утилита "Ёлочка", то что не попадает под ПДА/диалоги и лок.файл можно ей в процессе игры переводить.
https://perevod-teksta-s-ekrana.blogspot.com/

А для более точного перевода, с поиском синонимов рекомендую DeepL, там кликаешь по переведенному слову и он показывает варианты, иногда очень помогает и, зараза, учится. Если год назад он не понимал слова из игры, то сейчас более точен.

**Reply by SirMaxBlack** [reply]
О, спасибо большое! Изучу

**Comment by All4X4** [comment]
И снова 

_Lava Tubes,Lava Tubes,,,,,,,,Лавовая пещера,,,,,,,,
Lava Tubes,Lava Tubes,,,,,,,,Лавовый туннель,,,,,,,,_

Это **Лавовая Трубка**  реальное, устоявшееся название. Для упрощения называют туннелями. Но в файле именно трубка. Короче говоря, если "звёзды" называть "солнцами" - это глупо, тут тот самый случай.  На Земле, древние люди веками их использовали в качестве жилья. Когда речь идёт про Марс,  если найдут эти трубки там, то это будет большой удачей. 
 Авторы RE, часто используют устоявшиеся названия, кто-то из них шарит или кто-то подсказывает. 
 
 И ещё, 9 мая правил локфайл, при отправке произошел сбой GitHub Desktop. Пытался дважды реанимировать. И я так понимаю, эти правки не дошли. Сегодня попытался еще раз, если не пройдут, то ну его этот гитхаб.

**Reply by SnakeSel** [reply]
В данном случае я специально оставил туннель. Мне кажется более понятно о чем речь. Тем более что и трубка и туннель являются правильными названиями. [Wiki](https://ru.wikipedia.org/wiki/%D0%9B%D0%B0%D0%B2%D0%BE%D0%B2%D0%B0%D1%8F_%D1%82%D1%80%D1%83%D0%B1%D0%BA%D0%B0)

**Reply by All4X4** [reply]
Ну смотри сам. Если не устраивает реальное название, то дело хозяйское. ИМХО, игры должны не только развлекать, но расширять кругозор. В других играх, кстати,  это "трубка". )))
